apps/reward-manager/src/gas_oracle.ts

```diff
@@ -0,0 +1,63 @@
+import { BigNumber, providers } from "ethers";
```

Comment:
I don't think this is really used anymore, this uses the legacy gas price mechanism I believe

Reply:
will remove it completely since we can set max gas price in defender although currently no cap is set.